src/main/java/com/HuePlugin/HuePlugin.java

```diff
@@ -106,12 +114,20 @@ public void onChatMessage(ChatMessage chatMessage)
 			if(chatMessage.getMessage().equals("!Testhueconnection")){
 				if(connectionSucessful){
 					client.addChatMessage(ChatMessageType.GAMEMESSAGE,"","Your smart lights are connected.","");
-
 				}else{
 					client.addChatMessage(ChatMessageType.GAMEMESSAGE,"","Your smart lights are not connected.","");
-
 				}
+			}
+			if(chatMessage.getMessage().equals("!Testlights")) {
```

Comment:
With this addition you can remove the "!Testhueconnection" feature and have this as the sole feature to confirm your connection has been established.

Reply:
Yup, changed! Sounds good!